src/test/java/dk/cngroup/lentils/repository/HintRepositoryTest.java

```diff
@@ -89,4 +92,13 @@ public void getAllForCypherTest() {
 
         assertEquals(ObjectGenerator.NUMBER_OF_HINTS_FOR_CYPHER, hints.size());
     }
+
+    @Test(expected = javax.validation.ConstraintViolationException.class)
+    public void hintWithEmptyNameTest() {
+        Cypher cypher = cypherService.save(new Cypher(TEST_CYPHER_NAME, TESTED_STAGE, TEST_LOCATION, CODEWORD));
```

Comment:
Jeste premyslim, jestli by nestalo za to, tohle napsat tak, ze budes mit factory metodu, ktera ti vrati validni objekt.
V testu pak schvalne pokazis konkretni atribut a zkusis ulozit.

Takze kdyz se treba prida dalsi validace nebo atribut, nestane se ti, ze test pada z jinehodu duvodu a je tak vlastne false-positive. Nebo nebudes muset menit tolik mist, kde tu entitu takhle vytvaris.